core/domain/email_manager.py

```diff
@@ -1375,17 +1373,62 @@ def send_mail_to_notify_admins_suggestions_waiting_long(
         log_new_error('There were no admins to notify.')
         return
 
-    suggestion_descriptions = []
+    translation_suggestion_descriptions = []
+    question_suggestion_descriptions = []
     # Get the html for the list of suggestions that have been waiting too long
     # for a review.
     for reviewable_suggestion_email_info in reviewable_suggestion_email_infos:
-        suggestion_descriptions.append(
-            _create_html_for_reviewable_suggestion_email_info(
-                reviewable_suggestion_email_info))
-
-    list_of_suggestion_descriptions = ''.join(
-        suggestion_descriptions)
+        if (
+                feconf.SUGGESTION_TYPE_TRANSLATE_CONTENT ==
+                reviewable_suggestion_email_info.suggestion_type
+            ):
+            translation_suggestion_descriptions.append(
+                _create_html_for_reviewable_suggestion_email_info(
+                    reviewable_suggestion_email_info))
+        if (
+                feconf.SUGGESTION_TYPE_ADD_QUESTION ==
+                reviewable_suggestion_email_info.suggestion_type
+            ):
+            question_suggestion_descriptions.append(
+                _create_html_for_reviewable_suggestion_email_info(
+                    reviewable_suggestion_email_info))
+
+    list_of_translation_suggestion_descriptions = ''.join(
+        translation_suggestion_descriptions)
+    if list_of_translation_suggestion_descriptions:
+        user_ids = []
+        user_ids.extend(admin_ids)
```

Comment:
Maybe, this can be initialized with admin_ids outside the `if` condition, since it seems like both are using it.

Reply:
We need to clear `user_ids` inside both if blocks anyway because the emails pretaining to translations go to translation admins and emails related to questions go to the questions admin. So I don't think we need to move the initialisation outside the `if`.